### PR TITLE
fix: pass correct argument for otel-agent logging

### PR DESCRIPTION
**Pull request by sdowell**
The opentelemetry image does not accept -v as the argument for logging verbosity as the other containers do. This change updates it to set the appropriate argument using the zap logging library.

### CODE REVIEW

File: pkg/api/configsync/v1beta1/resource_override.go [hunk 1]
```diff
@@ -162,8 +162,8 @@ type ContainerLogLevelOverride struct {
 
 	// logLevel specifies the log level override value for the container.
 	// The default value for git-sync container is 5, while all other containers will default to 0.
```
Comment (nan-yu) [inline pkg/api/configsync/v1beta1/resource_override.go:164]: Is the default log level for otel-agent -1 or 0?

Comment (sdowell) [reply pkg/api/configsync/v1beta1/resource_override.go:164]: The default is 0, which maps to `INFO`

**Review by nan-yu — APPROVED** [review]
/lgtm

**Pull request merged**